pdb2sql/StructureSimilarity.py

```diff
@@ -485,24 +501,28 @@ def compute_lrmsd_pdb2sql(self, exportpath=None, method='svd'):
         backbone = ['CA', 'C', 'N', 'O']
 
         # create the sql
-        sql_decoy = pdb2sql(self.decoy, sqlfile='decoy.db')
-        sql_ref = pdb2sql(self.ref, sqlfile='ref.db')
+        sql_decoy = pdb2sql(
+            self.decoy, sqlfile='decoy.db', fix_chainID=True)
```

Comment:
Can you add a warning for using `fix_chainID` (in the `_fix_chainID` of `pdb2sqlcore.py`)? Because it's hard to make sure the chain orders are same for different PDB files.

Reply:
good point I'll do that :)

Reply:
I ended up checking the chains in the pdb and use them to compute the interface residue. It's a bit more clear I think